ecosystem/web-wallet/src/index.tsx

```diff
@@ -3,19 +3,19 @@
 
 import React from 'react'
 import ReactDOM from 'react-dom'
-import { BrowserRouter, Routes, Route } from 'react-router-dom'
+import { MemoryRouter, Routes, Route } from 'react-router-dom'
 import Wallet from './pages/Wallet'
 import Login from './pages/Login'
 import { GlobalStateProvider } from './GlobalState'
 
 ReactDOM.render(
   <GlobalStateProvider>
-    <BrowserRouter>
+    <MemoryRouter>
```

Comment:
BrowserRouter doesn't work within an extension

Reply:
Got it!